database.md

```diff
@@ -0,0 +1,36 @@
+# Database
+
+## Requirements
+Choose a Postgres client, [pgAdmin](https://www.pgadmin.org/) is the most full featured, intelliJ also has a built in plugin.
+
+### Connecting to Local Database
+1. Run the docker file ./docker-compose.postgres.yml in the root of the project
+2. Run the `generate_env.sh` script:
+
+    ```bash
+    ./generate_env.sh
+    ```
+
+3. Run the `./gradlew clean app:run` command in your terminal
+4. Configure your preferred db client to connect to the database with the connection information.  NOTE: The database runs on the non-default port of `5433` to avoid collision with the ReportStream Postgres database.
```

Comment:
Will the db connection information always be in `./docker-compose.postgres.yml` ?

Reply:
For local connections, yes